### DISCUSSION TITLE
Picking renderables with transparent blending

### DISCUSSION THREAD

**Discussion by grassydragon** [google/filament#5851] [Q&A]
Hi!
Is it possible to pick renderables that have `blending: transparent` in their material?
I've seen this discussion https://github.com/google/filament/discussions/5007#discussioncomment-1951280, however, maybe there is a way to enable picking for those renderables.

**Comment by pixelflinger** [comment]
Unfortunately, this is not possible at the moment. It could be implemented as a special picking pass, it's all feasible but not implemented and unfortunately not a high priority for us.

**Reply by grassydragon** [reply]
Thank you for your answer! Where do you think this feature should be enabled and disabled if we decide to have a go at implementing it?
